packages/react-charts/src/components/ChartDonutUtilization/ChartDonutUtilization.tsx

```diff
@@ -596,8 +596,12 @@ export const ChartDonutUtilization: React.FunctionComponent<ChartDonutUtilizatio
       const donutThresholds = getDonutThresholds();
       const mergeThemeProps = (i: number) => {
         // Merge just the first color of dynamic (blue, green, etc.) with static (gray) for expected colorScale
-        newTheme.pie.colorScale[0] = donutThresholds[i].color;
-        newTheme.legend.colorScale[0] = donutThresholds[i].color;
+        if (newTheme.pie.colorScale instanceof Array) {
```

Comment:
https://github.com/FormidableLabs/victory/commit/439e0ad8de23d1d6ed5580a90f60feaca3f09c4e#diff-1d0ded8dfdfbe48d50e6a99272c7548f81143f224f88b6528023cc87a32e9a40L317-R317
Here `colorScale` is no longer a `string` array, but the `ColorScalePropType`, which is `type ColorScalePropType =
  | "grayscale"
  | "qualitative"
  | "heatmap"
  | "warm"
  | "cool"
  | "red"
  | "green"
  | "blue"
  | string[];`. We can only use an index on Array so we need to decide whether it's an array. (Any better ideas?)

Reply:
We will never use "grayscale" | "qualitative" | "heatmap" | "warm" | "cool" | "red" | "green" | "blue" as our `colorScale`. These values only have meaning for the raw Victory charts.

We could change our `colorScale` types from `ColorScalePropType` to `string[]`. It's possible to omit those types from `ColorScalePropType`, but might be easier to override `ColorScalePropType` via src/typings/victory.d.ts?

Reply:
Created https://github.com/patternfly/patternfly-react/issues/5129 to address the `colorScale` proper type.